mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_marc_authority_job_profile.sql

```diff
@@ -22,3 +22,144 @@ INSERT INTO ${myuniversity}_${mymodule}.job_profiles (id, jsonb) values
+  "folioRecord": "AUTHORITY",
+  "childProfiles": [],
+  "parentProfiles": [],
+    "metadata": {
+      "createdDate": "2021-10-08T14:00:00.000",
+      "updatedDate": "2021-10-08T15:00:00.462+0000",
+      "createdByUserId": "00000000-0000-0000-0000-000000000000",
+      "updatedByUserId": "00000000-0000-0000-0000-000000000000"
+    },
+    "userInfo": {
+      "lastName": "System",
+      "userName": "System",
+      "firstName": "System"
+    }
+}')
+ON CONFLICT DO NOTHING;
+
+INSERT INTO ${myuniversity}_${mymodule}.mapping_profiles (id, jsonb) values
+  ('6a0ec1de-68eb-4833-bdbf-0741db25c314', '{
+  "id": "6a0ec1de-68eb-4833-bdbf-0741db25c314",
+  "name": "Default - Create authorities",
+  "deleted": false,
+  "metadata": {
+    "createdDate": "2021-10-08T14:00:00.000",
+    "updatedDate": "2021-10-08T15:00:00.462+0000",
+    "createdByUserId": "00000000-0000-0000-0000-000000000000",
+    "updatedByUserId": "00000000-0000-0000-0000-000000000000"
+  },
+  "userInfo": {
+    "lastName": "System",
+    "userName": "System",
+    "firstName": "System"
+  },
+  "description": "This field mapping profile is used with FOLIO''s default job profile for creating Inventory Authorities and SRS MARC Authorities records. It can be edited, duplicated, deleted, or linked to additional action profiles.",
+  "childProfiles": [],
+  "mappingDetails": {
+    "name": "authority",
+    "recordType": "AUTHORITY",
+    "mappingFields": [
+      {
+        "name": "personalName",
+        "path": "authority.personalName",
+        "value": "",
+        "enabled": "false",
+        "subfields": []
+      }, {
+        "name": "corporateName",
+        "path": "authority.corporateName",
+        "value": "",
+        "enabled": "false",
+        "subfields": []
+      }, {
+        "name": "meetingName",
+        "path": "authority.meetingName",
+        "value": "",
+        "enabled": "false",
+        "subfields": []
+      }, {
+        "name": "uniformTitle",
+        "path": "authority.uniformTitle",
+        "value": "",
+        "enabled": "false",
+        "subfields": []
+      }, {
+        "name": "topicalTerm",
+        "path": "authority.topicalTerm",
+        "value": "",
+        "enabled": "false",
+        "subfields": []
+      }, {
+        "name": "subjectHeadings",
+        "path": "authority.subjectHeadings",
+        "value": "",
+        "enabled": "false",
+        "subfields": []
+      }, {
+        "name": "geographicName",
+        "path": "authority.geographicName",
+        "value": "",
+        "enabled": "false",
+        "subfields": []
+      }, {
+        "name": "genreTerm",
+        "path": "authority.genreTerm",
+        "value": "",
+        "enabled": "false",
+        "subfields": []
+      }, {
+        "name": "identifiers",
+        "path": "authority.identifiers",
```

Comment:
```suggestion
        "path": "authority.identifiers[]",
```
I thinks it's also an array like notes